spec/support/examples_group/shared_examples_for_provisioning_state_machine.rb

```diff
@@ -1,6 +1,7 @@
 shared_examples_for "common rhev state machine methods" do
   it "#customize_destination" do
     allow(@task).to receive(:get_provider_destination).and_return(nil)
+    allow(@task).to receive(:destination_image_locked?).and_return(false)
```

Comment:
@borod108 @pkliczewski this will cause merge conflicts. Just inline those shared_examples into the specs. This will make backporting much easier...